boot/zephyr/main.c

```diff
@@ -541,12 +572,20 @@ void main(void)
 
     if (fih_not_eq(fih_rc, FIH_SUCCESS)) {
         BOOT_LOG_ERR("Unable to find bootable image");
+#ifdef CONFIG_MCUBOOT_INDICATION_LED
+        mcuboot_led_error();
+#endif
         FIH_PANIC;
     }
 
     BOOT_LOG_INF("Bootloader chainload address offset: 0x%x",
                  rsp.br_image_off);
 
+#ifdef CONFIG_MCUBOOT_INDICATION_LED
```

Comment:
Do these need ifdefs? Aren't they defined as empty when `CONFIG_MCUBOOT_INDICATION_LED` is not defined?

Reply:
Interesting, this appears to have entered the code on the first rebase a while ago. Thought I actually catched and fixed all.